apps/ledger-live-desktop/src/renderer/icons/Cic.jsx

```diff
@@ -0,0 +1,24 @@
+// @flow
```

Comment:
Remove flow comments ?

Reply:
There is a "eslint" problem in LLD when we remove "flow" ("Type annotations require valid Flow declaration").
(Medium term, we are going to move the icons from LLD/LLM to Core.)

Reply:
Flow indeed still in use on LLD unfortunately.